router/routes.js

```diff
@@ -2,36 +2,48 @@ require('dotenv').config();
 const mongoose = require('mongoose');
 const express = require('express');
 const questionnaireModel = require('../models/Mongoschema');
-const createQuestionnaire = require('../controller/questionnairecontroller');
+const questionnaireController = require('../controller/questionnairecontroller');
 
 const router = express.Router();
 
-const uri = process.env.DB_URI;
-
 // GET /health --> returns database status
```

Comment:
In order to add comments to the endpoints, follow the standard https://apidocjs.com/